From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI breakage for iwlwifi
Patch-mainline: Never, SLE15 only
References: bsc#1051510

The patch
  patches.drivers/iwlwifi-add-shared-clock-PHY-config-flag-for-some-de
broke kABI due to a new field in iwl_cfg.
Protect it in a standard trick.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/net/wireless/intel/iwlwifi/iwl-config.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/wireless/intel/iwlwifi/iwl-config.h
+++ b/drivers/net/wireless/intel/iwlwifi/iwl-config.h
@@ -394,7 +394,9 @@ struct iwl_cfg {
 	u8 max_vht_ampdu_exponent;
 	u8 ucode_api_max;
 	u8 ucode_api_min;
+#ifndef __GENKSYMS__
 	u32 extra_phy_cfg_flags;
+#endif
 };
 
 /*
